controller/nodegroup.go

```diff
@@ -115,6 +123,7 @@ func buildLaunchTemplateData(group eksv1.NodeGroup, ec2Service *ec2.EC2) (*ec2.R
 			},
 		},
 		TagSpecifications: utils.CreateTagSpecs(group.ResourceTags),
+		SecurityGroupIds: securityGroupIds,
```

Comment:
Instead of creating a new slice manually, you can use `aws.StringSlice(securityGroups)` here directly.